README.md

```diff
@@ -2,10 +2,15 @@
 # Alfresco Content Services Deployment
 
 ## Contributing guide
-Please use [this guide](CONTRIBUTING.md) to make a contribution to the project.
+Please use [this guide](CONTRIBUTING.md) to make a contribution to the project and information to report any issues.
 
 This project contains the code for starting the entire Alfresco Content Services (Enterprise) product with **Docker** or **Kubernetes**.
 
+The **master** branch of this repository will endeavour to support below deployments:
+- [Docker Compose](docker-compose/docker-compose.yml) (latest)
```

Comment:
[Docker Compose](docker-compose/docker-compose.yml) (latest): For development and trials
MiniKube (latest): For development and POCs
[Helm - AWS Cloud with Kubernetes](helm/alfresco-content-services) (latest): For production or as a basis of production deployments